lib/services/page-service.ts

```diff
@@ -0,0 +1,152 @@
+import util from "../shared/util";
+
+// Pages backup data
+const BACKUP = require("../../consts/pages-backup-data");
+
+export class PageService implements IPageService {
+    getPages() {
+        return new Promise((resolve, reject) => {
+            resolve(BACKUP.fallback);
+        });
+    }
+
+    getFlavor(appPath: string) {
```

Comment:
Isn't this a characteristic of the app? It seems to me a better place for this method would be the template service.

Reply:
As a matter of fact it is, and in my opinion it should be handled by Fusion. However, we do plan to reuse it across the extension so for me it makes more sense to be in the Util Class

Reply:
To be discussed in the afternoon meeting.